tests/Integration/StructuredData/SinglePostTest.php

```diff
@@ -652,7 +652,8 @@ public function test_post_with_categories_as_tags_without_categories(): void {
 		$parsely_options['cats_as_tags'] = true;
 		update_option( 'parsely', $parsely_options );
 
-		wp_remove_object_terms( $post_id, 'uncategorized', 'category' );
+		$default_category_name = $parsely->get_default_post_category_name();
+		wp_remove_object_terms( $post_id, $default_category_name, 'category' );
```

Comment:
The second argument here is using the slug (but it could also use the ID), but you're now passing in the category name, not the category slug.

Reply:
You are correct. I've fixed this.